packages/orbit-components/src/Seat/components/primitives/TransitionPathStroke.tsx

```diff
@@ -0,0 +1,35 @@
+import * as React from "react";
+import cx from "clsx";
+
+import { TYPES } from "../../consts";
+import type { Type } from "../../types";
+
+type Props = {
+  type: Type;
+  selected: boolean;
+} & React.SVGProps<SVGPathElement> &
+  React.PropsWithChildren;
+
+const TransitionPathStroke = ({ children, type, selected, ...props }: Props) => {
+  return (
+    <path
+      className={cx(
+        "duration-fast transition-colors ease-in",
+        type === TYPES.LEGROOM && (selected ? "stroke-blue-normal" : "stroke-blue-light-active"),
+        type === TYPES.DEFAULT &&
+          (selected ? "stroke-product-normal" : "stroke-product-light-active"),
+        type === TYPES.UNAVAILABLE && "stroke-cloud-light-active",
+        selected &&
+          (type === TYPES.LEGROOM
+            ? "group-focus-visible:stroke-blue-normal-hover group-active:stroke-blue-normal-hover"
+            : "group-focus-visible:stroke-product-normal-hover group-active:stroke-product-normal-hover"),
```

Comment:
Can't this also be included in the conditions above? On the `true` part of each ternary from the LEGROOM and DEFAULT conditions

Reply:
It can, updated. Thanks!